nsv13/code/modules/overmap/types.dm

```diff
@@ -195,6 +199,10 @@
 	SSstar_system.bounty_pool += bounty //Adding payment for services rendered
 	. = ..()
 
+/obj/structure/overmap/nanotrasen/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/grav_taker) //ships are affected by gravity
```

Comment:
Or are you just trying to make Ai ships ignore gravity?

Reply:
yes, for now

Reply:
Their AI wont be able to handle it.